doc/Data Plane Code Generation/AzureSDKCodeGeneration_DataPlane_Quickstart.md

```diff
@@ -1,201 +1,168 @@
 # Azure SDK Code Geneneration Quickstart Tutorial (Data Plane)
 The purpose of the Azure SDK is to provide a unified developer experience across Azure services. In this tutorial, we will walk through creating a new Azure SDK Generated Client library for a data plane Azure service.  The output library will have a standardized API based on the  [.NET Azure SDK Design Guidelines](https://azure.github.io/azure-sdk/dotnet_introduction.html).
 
-We will use the [Azure SDK Generated Client template](https://github.com/Azure/azure-sdk-for-net/tree/main/sdk/template-dpg) project as a reference. We have a template [swagger file](https://github.com/Azure/azure-sdk-for-net/tree/main/sdk/template-dpg/Azure.Template.Generated/src/swagger) containing a REST API definition based on [Azure REST API Guidelines](https://github.com/microsoft/api-guidelines/blob/vNext/azure/Guidelines.md). We will use the swagger to generate SDK using [autorest.csharp](https://github.com/Azure/autorest.csharp).
+We will use the swagger file which contains a REST API definition based on [Azure REST API Guidelines](https://github.com/microsoft/api-guidelines/blob/vNext/azure/Guidelines.md). We will use the swagger to generate SDK using [autorest.csharp](https://github.com/Azure/autorest.csharp).
 
 **Learn more**: to understand more about Azure SDK Code Geneneration, see the [Azure SDK Code Geneneration docs](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/core/Azure.Core/samples/ProtocolMethods.md).
 
 This tutorial has following sections:
 
-- [Prerequisites](#prerequisites)
-- [Setup your repo](#setup-your-repo)
-- [Getting started](#getting-started)
-  - [Create the project](#create-the-project)
-  - [Create the generated layer](#create-the-generated-layer)
-- [Tests](#tests)
-- [Samples](#samples)
-- [Snippets](#snippets)
-- [Export Public API](#export-public-api)
-- [README](#readme)
-- [Changelog](#changelog)
+- [Azure SDK Code Geneneration Quickstart Tutorial (Data Plane)](#azure-sdk-code-geneneration-quickstart-tutorial-data-plane)
+  - [Prerequisites](#prerequisites)
+  - [Setup your repo](#setup-your-repo)
+  - [Create starting package](#create-starting-package)
```

Comment:
Would you be willing to change this to "starter package"?

Reply:
updated